.github/workflows/auto_approve_pull_requests.yml

```
@@ -0,0 +1,17 @@
+
+jobs:
+  auto-approve:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    if: github.actor == 'rocketmq-rust-bot'
+    steps:
+      - uses: hmarr/auto-approve-action@v4
+        with:
+          github-token: ${{ secrets.BOT_TOKE }}
+          review-message: "LGTM"
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)